issue_release_status.go

```diff
@@ -131,41 +131,22 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 
 	logger.Debug().Msgf("%s", botMessage)
 
-	/*
-		var path string
-		var resp httpinternal.ListPoliciesResponse
-		var client *httpinternal.Client
-		var serviceName string
-
-		params := url.Values{}
-		params.Add("service", serviceName)
-
-		path, err := client.URLWithQuery(path, params)
-		if err != nil {
-			return err
-		}
-
-		err = client.Do(http.MethodGet, path, nil, &resp)
-	*/
-
 	// Send PR comment
-
-	/*client, err := handler.NewInstallationClient(installationID)
+	client, err := handler.NewInstallationClient(installationID)
 	if err != nil {
-		return errors.Wrap(err, "installing client")
+		return errors.Wrap(err, "creating new github.Client")
```

Comment:
As the installation ID is derived from the incoming event, it might be nice to have that in case of errors?

```suggestion
		return errors.Wrapf(err, "creating new github.Client from installation id '%s'", installationID)
```

Reply:
Good idea